Assets/Scripts/Models/DialogBoxInformation/DialogComponent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+using UnityEngine;
+
+public class DialogComponent
+{
+    [XmlAttribute("Type")]
+    public string ObjectType;
+
+    [XmlElement]
```

Comment:
[XmlAttribute("Position")]
shame here that Unity doesn't serialize this with x,y,z as attributes. Though use can create wrapper class, adjust the format then have it return Unitys Vector3..